doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -263,6 +257,58 @@ returns -1.
 SSL_get_negotiated_group() returns the NID of the negotiated group used for
 key exchange, or NID_undef if there was no negotiated group.
 
+=head1 EXAMPLES
+
+Assume the server I<list> is "P-521:P-256/P-384/X25519:ffdhe2048" and client
+I<list> is "P-521:*P-384" when connecting to such a server, meaning that the
+client supports C<P-521> but does not send a key share for this group to the
+server, and the client supports C<P-384> including key share for this group.
+With both server and client preference, an HRR will be triggered for C<P-521>
+despite the availability of a key share for P-384, which overlaps with a lower
+priority server-side tuple.
```

Comment:
The examples are not substantially better. No major issues. I just feel that with the examples below, that really cover the issues well, the above "concrete" example isn't nearly as good, and the presence of specific groups that may or may not be the ones used in some scenario is more of a distraction, so I'd drop this one too...  Perhaps others will have a different view...

Reply:
[ Edited from a  typo that had "not" instead of "now". ]